### PR TITLE
[bgp]: Implement Universal Traffic Shift for SONiC

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**- What I did**
Implemented Universal Traffic Shift for SONiC which allows us to remove traffic from SONiC device with minimal packet drop.

**- How I did it**
1. Created two base route-maps: TO_BGP_PEER_V4 and TO_BGP_PEER_V6 which are applied to all active peers automatically. The route-maps by default enable everything.
2. Created two prefix-lists: PL_LoopbackV4 and PL_LoopbackV6 which filter our loopback address only.
3. Created scripts:
  - TSA - to remove traffic from the box by inserting two extra rules to the route-maps:
       - advertise only loopback prefix
       - don't advertise anything else
  - TSB - to remove rules after TSA
  - TSC - to check state of the rules
4. expose TSA, TSB, TSC scripts to the base system

**- How to verify it**
Build the image.
Use TSA, TSB, and TSC scripts and check what updates your DUT send to its bgp neighbors.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-fpm-frr/bgpd.conf.default.j2 [hunk 1]
```diff
@@ -7,6 +7,21 @@ route-map FROM_BGP_SPEAKER_V4 permit 10
...
+{% elif prefix | ipv6 and name == 'Loopback0' %}
+ipv6 prefix-list PL_LoopbackV6 permit {{ prefix | ip }}/64
+{% endif %}
+{% endfor %}
+!
+{# generate default peer route-maps #}
+!
+route-map TO_BGP_PEER_V4 permit 100
+!
+route-map TO_BGP_PEER_V6 permit 100
```
Comment (lguohan) [inline dockers/docker-fpm-frr/bgpd.conf.default.j2]: lack of sonic-cfggen test.

**Comment by pavel-shirshov** [comment]
retest vs please

**Comment by pavel-shirshov** [comment]
retest vsimage please

**Comment by pavel-shirshov** [comment]
retest vs please

**Comment by pavel-shirshov** [comment]
retest vs please

**Pull request merged**